Remove Empty MRU Warning

ssigwart opened the pull request:
When using find in project, the new Include filter results in the following error log message:
```
[WARNING] ko.mru: MRU_add: warning: no entry: prefName='find-includeFiletypesMru', entry=''
```
It's normal that an empty string should be saved there.

th3coop commented:
I agree in this case an empty MRU entry makes sense but in many other cases it does not.  Looks like find2.p.js is the culprit.  I don't know why but at line 641 it's checking if it's "find in files", which looks like "find in project" falls under that, and if it is then it saves the includes list but leaves off the `allowBlank` arg.  I don't understand why you would want that...*checks git blame*...11 years and counting.  

I'd personally be fine with just move that code in find2.p.js so that includes are still saved as blank if used blank.

ssigwart commented:
I think it might be using the `koIFindContext.FCT_IN_COLLECTION` option just below that code in `find2.p.js` for project searches.  I added the `allowBlank` option earlier this year, but didn't update the find in files option.  I assume `if (widgets.includes.value)` was to prevent this error, but now that there's `allowBlank`, I update it to use that.  I also updated it to pass `allowBlank` into `MRU_add` so I can ignore the error in only those cases.
